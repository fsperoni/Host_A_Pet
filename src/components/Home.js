import React from 'react';
import pets from '../assets/pets.jpeg';

const Home = () => {
  return (
    <div className="container mt-3">
      <h1 className="text-center">Host a Pet!</h1>
      <p className="text-center" >
        We connect pet owners and pet hosts for amazing experiences. Sign up today
        and either search for a pet to host or someone to host your pet. You can browse
        by distance from your postal code, or filter results by host and pet owner
        ratings.
      </p>
      <p>Review hostings, assigning a score, or go through previous hostings 
        feedback to help you choose the best host for your pet, or the best pet for you
        to host.
      </p>
      <img className="img-fluid rounded mx-auto d-block" src={pets} />
    </div>
  )
}

export default Home;