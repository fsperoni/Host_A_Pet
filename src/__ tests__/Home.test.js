import { render } from "@testing-library/react";
import Home from "../components/Home";

// smoke test
it('renders without crashing', function() {
  render(<Home />);

});

//snapshot test
it("matches snapshot", function() {
  const {asFragment} = render(<Home />);
  expect(asFragment()).toMatchSnapshot();
});

it("should have 'Host a Pet!' in the component.", function() {
  const { getByText } = render(<Home />);
  expect(getByText("Host a Pet!")).toBeInTheDocument();
});