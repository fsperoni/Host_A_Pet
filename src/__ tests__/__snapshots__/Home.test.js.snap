// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container mt-3"
  >
    <h1
      class="text-center"
    >
      Host a Pet!
    </h1>
    <p
      class="text-center"
    >
      We connect pet owners and pet hosts for amazing experiences. Sign up today and either search for a pet to host or someone to host your pet. You can browse by distance from your postal code, or filter results by host and pet owner ratings.
    </p>
    <p>
      Review hostings, assigning a score, or go through previous hostings feedback to help you choose the best host for your pet, or the best pet for you to host.
    </p>
    <img
      class="img-fluid rounded mx-auto d-block"
      src="pets.jpeg"
    />
  </div>
</DocumentFragment>
`;
